test/WebSites/BasicWebSite/StartupWithGlobalRouting.cs

```diff
@@ -31,9 +31,9 @@ public void Configure(IApplicationBuilder app)
 
             app.UseGlobalRouting();
 
-            app.UseMvcWithEndpoint(routes =>
+            app.UseMvc(routes =>
```

Comment:
So what will this do? Since `options.UseGlobalRouting` isn't set.

Reply:
Good catch. It will use just normal routing.

I was thinking yesterday that a test needs to be added to functional to double check that global routing is running because the code and behavior is so close. A good problem to have 😬

Reply:
It is using global routing now because `CompatibilityVersion.Latest`.

I also added some tests to each functional project that verifies whether vNext routing is or is not running.